BoostIO/BoostNote.next
# Feature/add code fence buttons to cloud markdown

**Add code fence to cloud**

- Refactor CodeFence and rehype attacher to shared components
- Add load mode function to shared codemirror util

Tested in:
Webapp dev (cloud space)
Desktop app dev (cloud and local space)

Tests:
- try saving snippet (works)
- try copy/pasting snippet (works)
- export note with snippets (HTML, PDF, Markdown)
  - works but previous bug with incorrect styles for cloud space (PDF export only) is making code blocks invisible (white text, white background) - should be fixed in another PR
